docs/scraped-metrics.md

```diff
@@ -38,157 +38,114 @@ The following table contains information about metrics scraped by Sumo Logic's P
-| `kube_pod_container_status_restarts_total`               | kube-state-metrics | yes       |
-| `kube_pod_container_status_terminated_reason`            | kube-state-metrics | yes       |
-| `kube_pod_container_status_waiting_reason`               | kube-state-metrics | yes       |
-| `kube_pod_status_phase`                                  | kube-state-metrics | yes       |
-| `kube_pod_info`                                          | kube-state-metrics | yes       |
-| `kube_service_info`                                      | kube-state-metrics | yes       |
-| `kube_service_spec_external_ip`                          | kube-state-metrics | yes       |
-| `kube_service_spec_type`                                 | kube-state-metrics | yes       |
-| `kube_service_status_load_balancer_ingress`              | kube-state-metrics | yes       |
-| `node_cpu_seconds_total`                                 | node-exporter      | yes       |
-| `node_load1`                                             | node-exporter      | yes       |
-| `node_load5`                                             | node-exporter      | yes       |
-| `node_load15`                                            | node-exporter      | yes       |
-| `node_disk_io_time_weighted_seconds_total`               | node-exporter      | yes       |
-| `node_disk_io_time_seconds_total`                        | node-exporter      | yes       |
-| `node_vmstat_pgpgin`                                     | node-exporter      | yes       |
-| `node_vmstat_pgpgout`                                    | node-exporter      | yes       |
-| `node_memory_MemFree_bytes`                              | node-exporter      | yes       |
-| `node_memory_MemAvailable_bytes`                         | node-exporter      | yes       |
-| `node_memory_Cached_bytes`                               | node-exporter      | yes       |
-| `node_memory_Buffers_bytes`                              | node-exporter      | yes       |
-| `node_memory_MemTotal_bytes`                             | node-exporter      | yes       |
-| `node_network_receive_drop_total`                        | node-exporter      | yes       |
-| `node_network_transmit_drop_total`                       | node-exporter      | yes       |
-| `node_network_receive_bytes_total`                       | node-exporter      | yes       |
-| `node_network_transmit_bytes_total`                      | node-exporter      | yes       |
-| `node_filesystem_avail_bytes`                            | node-exporter      | yes       |
-| `node_filesystem_size_bytes`                             | node-exporter      | yes       |
-| `node_filesystem_files_free`                             | node-exporter      | yes       |
-| `node_filesystem_files`                                  | node-exporter      | yes       |
+| name                                              | source             | forwarded |
```

Comment:
This table now only includes metrics that are forwarded to Sumo Logic. So now the "forwarded" column is not useful. (Before there was only one metric mentioned that was not forwarded.) Maybe we should remove it.

Reply:
yes I agree, if its scraped, we send it by default.